### DISCUSSION TITLE
Recommended scheduler for DEIS sampling?

### DISCUSSION THREAD

**Discussion by not-ski** [comfyanonymous/ComfyUI#3887] [Q&A]
Refer to commit https://github.com/comfyanonymous/ComfyUI/commit/44947e7ad489eaadb38007ca7aabd71aef7fa10e for implementation details.

I read the diff-solvers documentation, which seems to suggest the usage of "time_uniform" (`sgm_uniform`?) with DEIS sampling.

@qsh-zh, as one of the authors behind DEIS, what would you suggest? Thanks in advance <3

**Comment by cdxOo** [comment]
When trying DEIS it worked well with the "beta" scheduler @ CFG 7.0
